lightningd/plugin.c

```diff
@@ -1014,6 +1019,8 @@ static void plugin_config(struct plugin *plugin)
 	json_add_string(req->stream, "rpc-file", ld->rpc_filename);
 	json_object_end(req->stream);
 
+	json_add_bool(req->stream, "startup", plugin->plugins->startup);
```

Comment:
Adding startup like this will make all existing plugins incompatible since pylightning unpacks the arguments, and adding a new one will cause it to get an unknown argument. Can we make this part of the `configuration`?

Reply:
Yes sure, anyway I made a fixup afterwards :-). I can remove the fixup commit and make this field part of the `configuration`.